packages/cozy-jobs-cli/package.json

```diff
@@ -9,12 +9,10 @@
   },
   "license": "MIT",
   "bin": {
-    "cozy-konnector-dev": "bin/cozy-konnector-dev",
-    "cozy-konnector-standalone": "bin/cozy-konnector-standalone",
-    "cozy-konnector-shell": "bin/cozy-konnector-shell",
-    "cozy-run-shell": "bin/cozy-run-shell",
-    "cozy-run-standalone": "bin/cozy-run-standalone",
-    "cozy-run-dev": "bin/cozy-run-dev"
+    "cozy-konnector-shell": "src/konnector-shell",
+    "cozy-konnector-dev": "src/konnector-dev",
+    "cozy-konnector-standalone": "src/konnector-standalone",
```

Comment:
We should keep at least cozy-run-standalone as deprecated link to src/konnector-standalone.js to avoid change in all the existing connectors.